base/applications/shutdown/lang/pl-PL.rc

```diff
@@ -29,7 +29,7 @@ BEGIN
     CHECKBOX "Planowane", IDC_PLANNED, 196, 155, 51, 10, BS_AUTOCHECKBOX | WS_TABSTOP
     COMBOBOX IDC_REASON_CODE, 21, 168, 226, 105, WS_TABSTOP | CBS_DROPDOWN
     LTEXT "Ko&mentarz:", IDC_COMMENT_CAPTION, 21, 205, 226, 8
-    EDITTEXT IDC_COMMENT_TEXT, 21, 226, 216, 20, WS_VSCROLL
+    EDITTEXT IDC_COMMENT_TEXT, 21, 216, 226, 22, WS_VSCROLL
```

Comment:
Looking more generally at this `IDD_GUI` dialog:
https://git.reactos.org/?p=reactos.git&a=search&h=c507b701ab8a64e4bfbce2b304043e75afbc6283&st=grep&s=IDD_GUI
is it possible to uniformize the polish dialog with the sizes/positions of elements used for it in the other languages? Or would this cause problems fitting the text?